pkg/ignition/ignition_test.go

```diff
@@ -0,0 +1,155 @@
+package ignitionutil
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/go-openapi/swag"
+	. "github.com/onsi/ginkgo/v2/dsl/core"
+	. "github.com/onsi/gomega"
+	assetignition "github.com/openshift/installer/pkg/asset/ignition"
+)
+
+const (
+	fakeIgnition32 = `{
+		"ignition": {
+		  "config": {},
+		  "version": "3.2.0"
+		},
+		"storage": {
+		  "files": []
+		}
+	  }`
+
+	fakeIgnition99 = `{
+		"ignition": {
+		  "config": {},
+		  "version": "9.9.0"
+		},
+		"storage": {
+		  "files": []
+		}
+	  }`
+)
+
+type FakeOS struct{}
+
+func (FakeOS) MkdirTemp(dir, prefix string) (string, error) {
```

Comment:
nit for later: looks mostly similar to the struct in release_test. Maybe we can try to extract it later on. So we could override only the needed funcs.